src/main/java/com/monumental/models/Reference.java

```diff
@@ -12,9 +12,6 @@
  */
 
 @Entity
-@Table(name = "reference", uniqueConstraints = {
-        @UniqueConstraint(columnNames = "id")
```

Comment:
Don't need this anymore due to `AbstractAuditable`

Reply:
Ah, didn't see this before.